proparse/src/main/java/org/prorefactor/proparse/antlr4/ProgressLexer.java

```diff
@@ -369,7 +453,7 @@ private int escape() throws IOException {
   }
 
   private String getFilename() {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Rename method "getFilename" to prevent any misunderstanding/clash with method "getFileName" defined on line 146. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1845)